sessionctx/variable/statusvar.go

```diff
@@ -14,14 +14,16 @@
 package variable
 
 import (
+	"bytes"
+	"crypto/tls"
+
 	"github.com/juju/errors"
 )
 
 var statisticsList []Statistics
-var globalStatusScopes = make(map[string]ScopeFlag)
 
-// DefaultScopeFlag is the status default scope.
-var DefaultScopeFlag = ScopeGlobal | ScopeSession
+// DefaultStatusVarScopeFlag is the default scope of system variables.
+var DefaultStatusVarScopeFlag = ScopeGlobal | ScopeSession
```

Comment:
To clarify that it is for status variables.
When referencing this flag, old: `variable.DefaultScopeFlag`, new: `variable.DefaultStatusVarScopeFlag`.